apis/services/userService.js

```diff
@@ -2,36 +2,90 @@ const { userDao } = require('../models');
 const bcyrpt = require('bcrypt');
 const jwt = require('jsonwebtoken');
 
+
+const signUp = async (userName, password, fullName, phoneNumber, address, birth, gender) => {
+
+
+    const today = new Date();
+    const todayTime = today.getTime();
+    const birthTime = new Date(birth);
+    let age = Math.floor((todayTime - birthTime) / (1000 * 60 * 60 * 24 * 365));
+    if (age < 15) {
+        const err = new Error("만 14세 이하는 회원가입 불가")
+        err.statusCode = 401;
+        throw err
+    }
```

Comment:
- 제한 나이도 상수로 지정해서 사용해 주세요.
- error message는 영어로 작성해 주세요.
- status code도 수정해 주세요.
```suggestion
    const birthTime = new Date(birth);
    const limitAge = 15
    let age = Math.floor((todayTime - birthTime) / (1000 * 60 * 60 * 24 * 365));
    
    if (age < limitAge) {
        const err = new Error(`Under ${limitAge} years old`)
        err.statusCode = 400;
        throw err
    }
```